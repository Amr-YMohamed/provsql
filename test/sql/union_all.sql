\set ECHO none
SET search_path TO public,provsql;

CREATE TABLE union_all_result AS
SELECT *,formula(provenance(),'personal_name') FROM (
  SELECT classification FROM personal WHERE city='Paris'
  UNION ALL
  SELECT value FROM personal_level
) t;

SELECT remove_provenance('union_all_result');
SELECT * FROM union_all_result ORDER BY classification;
DROP TABLE union_all_result;
