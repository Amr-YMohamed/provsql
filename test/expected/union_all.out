\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

 classification | formula  
----------------+----------
 unclassified   | John
 restricted     | Paul
 restricted     | Nancy
 restricted     | Nancy
 confidential   | Dave
 confidential   | Dave
 secret         | Ellen
 secret         | Susan
 top_secret     | Magdalen
 top_secret     | Magdalen
(10 rows)

